pkg/agent/core/ngt/handler/grpc/object_test.go

```diff
@@ -1337,6 +1337,118 @@ func Test_server_StreamGetObject(t *testing.T) {
 	}
 }
 
+func Test_server_GetTimestamp(t *testing.T) {
```

Comment:
Is there a reason you are implementing this test without using a vald test template? 🤔

Reply:
Because in the handler test, not only the arguments of the function call, but also the flow of the preliminary setup is part of the test. We have already agreed on this, and some tests in this file are implemented in the same way.